lms/djangoapps/verify_student/views.py

```diff
@@ -946,6 +947,32 @@ def _send_email(user_id, subject, message):
     user.email_user(subject, message, from_address)
 
 
+def _set_requirement_status(attempt, namespace, status, reason=None):
+    """Set requirement status and the reason of status for given user in
+    attempt.
+    """
+    checkpoint = None
+    try:
+        checkpoint = VerificationCheckpoint.objects.get(photo_verification=attempt)
+    except VerificationCheckpoint.DoesNotExist:
```

Comment:
Here and below, you're catching exceptions that will prevent the user's requirement status from being set, but don't return any value (e.g., a Boolean) that would help notify calling code of the failure.